dist/libexec-native-overrides/cli-common-platform.ps1

```diff
@@ -0,0 +1,9 @@
+function Get-ScalaCLIBinaryName {
+    if ($IsWindows) {
```

Comment:
This now requires PowerShell >= 6.0

See: https://learn.microsoft.com/en-us/previous-versions/powershell/scripting/whats-new/what-s-new-in-powershell-core-60?view=powershell-6#engine-updates

Reply:
Which released in 2023, is that too new?

Reply:
2018: https://github.com/PowerShell/PowerShell/releases/tag/v6.0.0

> is that too new?

Mmmh, the issue with this is that some scripts might not work on some machines without installing pwsh >= 6.0

Reply:
oh ok, the blog you says written 2023, so my bad